charts/chronicle/templates/tests/api-test.yaml

```diff
@@ -8,16 +9,99 @@ metadata:
     "helm.sh/hook": test
     "helm.sh/hook-delete-policy": hook-succeeded
 spec:
+  backoffLimit: 0
   template:
     spec:
-      containers:
-        - name: test
-          {{- include "lib.image" (dict "imageRoot" .Values.test.image "global" .Values.global ) | nindent 10 }}
-          command: {{ .Values.test.command }}
+      restartPolicy: Never
+      serviceAccountName:  {{ include "lib.serviceAccountName" . }}
+      automountServiceAccountToken: true
+      volumes: {{- include "lib.volumes" .Values.opa.tp.extraVolumes | nindent 8 }}
+        - name: shared-data
+          emptyDir: {}
+      initContainers:
+      {{- if .Values.auth.required }}
+        - name: wait-for-id-provider
+          {{- include "lib.image" (dict "imageRoot" .Values.test.api.image "global" .Values.global ) | nindent 10 }}
+          command: [ "sh", "-ec" ]
+          args:
+            - |
+              echo "Waiting for id-provider to be ready ..."
+              wait-for-it $HOST:$PORT --timeout=0
+              echo "Id-provider is ready. Exiting."
           env:
             - name: HOST
-              value: {{ include "chronicle.service.name" . }}-chronicle-api
+              value: {{ include "chronicle.id-provider.service" . }}
```

Comment:
weird here to use an environment var set to the template rather than just using the template